spec/presenters/bank_transaction_presenter_spec.rb

```diff
@@ -12,7 +12,7 @@
     subject(:headers) { described_class.headers }
 
     it { is_expected.to be_a Array }
-    it { expect(headers.count).to eq 9 }
+    it { expect(headers.count).to eq 10 }
```

Comment:
Just a heads up that I've added a couple of extra columns to this report in another [PR](https://github.com/ministryofjustice/laa-apply-for-legal-aid/pull/1744) so, depending on which of us merges to master first, you might need to rebase and change the value here.

Reply:
thank you, I'll make the updates after you merge